src/main/kotlin/com/bardsoftware/backend/fleet/rmarkdown/Publisher.kt

```diff
@@ -56,4 +52,15 @@ class Publisher(private val topicName: String) {
             publisher?.shutdown()
         }
     }
+
+    companion object {
+        fun createMessage(message: String): PubsubMessage? {
+            val data = ByteString.copyFromUtf8(message)
+            val pubsubMessage = PubsubMessage.newBuilder()
```

Comment:
Just write `return` here instead of local variable

Reply:
Done